README.md

```diff
@@ -314,3 +117,16 @@ It's important to note that all Valhalla logs for one-shot mode are piped to `st
 ### Batch Script Tool
 
 - [Batch Run_Route](./run_route_scripts/README.md)
+
+## Related projects
```

Comment:
i like this. its been missing for a long time. i cant help but thing there is more we should put here but at the moment i just cant put my finger on what is missing.. anyway thanks for starting the list!